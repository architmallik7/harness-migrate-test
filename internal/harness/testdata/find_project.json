{
    "status": "SUCCESS",
    "data": {
        "project": {
            "orgIdentifier": "default",
            "identifier": "pranavt1",
            "name": "pranav",
            "color": "#0063f7",
            "modules": [
                "CV",
                "CI",
                "STO",
                "CD",
                "TEMPLATESERVICE",
                "PMS",
                "CE",
                "CF",
                "CORE"
            ],
            "description": "",
            "tags": {}
        },
        "createdAt": null,
        "lastModifiedAt": 1663774477071
    },
    "metaData": null,
    "correlationId": "5bee8180-8609-4dfa-988f-2bc3c82ba94f"
}