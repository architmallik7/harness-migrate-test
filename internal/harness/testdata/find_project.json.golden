{
  "orgIdentifier": "default",
  "identifier": "pranavt1",
  "name": "pranav",
  "color": "#0063f7",
  "modules": [
    "CV",
    "CI",
    "STO",
    "CD",
    "TEMPLATESERVICE",
    "PMS",
    "CE",
    "CF",
    "CORE"
  ]
}